testing/MLDB-409-transform-dataset.js

```diff
@@ -64,7 +64,8 @@ var transform_config = {
             named: "rowName() + '_transformed'",
             limit: 3
         },
-        outputDataset: { id: 'transformed', type: 'sparse.mutable' }
+        outputDataset: { id: 'transformed', type: 'sparse.mutable' },
+        runOnCreation : false
```

Comment:
runOnCreation is true by default.  This test was running the procedure twice because it explicitly creates a procedure run after creating the procedure.  This was not causing any problem but I decided to fix it anyway.